src/controllers/auth.controller.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  JsonController,
+  Body,
+  Post,
+  BodyParam,
+  UnauthorizedError,
+  BadRequestError,
+  Res
+} from 'routing-controllers';
+import { getRepository } from 'typeorm';
+import * as _ from 'lodash';
+import { User } from '@entities/user.entity';
+import { createJWT } from '@services/jwt.service';
+
+@JsonController('/auth')
+export class AuthController {
+  @Post('/signup')
+  async signUp(@Body({ validate: false }) user: User, @Res() response: any) {
+    const newUser = await getRepository(User).save(user);
+    return response.send(_.omit(newUser, ['password']));
+  }
+
+  @Post('/signin')
+  async signIn(
+    @BodyParam('email') email: string,
+    @BodyParam('password') password: string
+  ) {
+    if (!email || !password) {
+      throw new BadRequestError('Missing params on body');
+    }
+
+    let user: User;
+    try {
+      user = await User.findOneOrFail({ where: { email } });
+    } catch (error) {
+      throw new UnauthorizedError('Invalid credentials');
+    }
+
+    //Check if encrypted password match
+    if (!user.comparePassword(password)) {
```

Comment:
Can't you just write this stuff in one line? 
`if (!user.comparePassword(password)) throw new UnauthorizedError('Invalid credentials');`

Reply:
And maybe using I18n (we can add it later)

Reply:
We should define (if it's not already done on the google's eslint) a max length for a line @GuilleLeopold . I like to use i18 but as you said, but that's work for another PR

Reply:
that one is probably because of the formatting limit of 80 col. per line

Reply:
Can we share this .vscode/config so all of us are aligned with the same?

Reply:
This "rule" is already present on prettier